spec.bs

```diff
@@ -71,7 +66,7 @@ dictionary AuctionAdInterestGroup {
   record<USVString, double> priorityVector;
   record<USVString, double> prioritySignalsOverrides;
 
-  WorkletExecutionMode executionMode = "compatibility";
+  USVString executionMode = "compatibility";

```

Comment:
Can this be a DOMString instead?

Reply:
Maybe? We've used USVString throughout FLEDGE, but I think we only really need it for URLs?

In this case, we just need to match whether the string equals "compatibility", or "group-by-origin" (and until support is removed, the deprecated "groupByOrigin"). So, we don't really need to do any parsing, like we do for URLs and origin strings.

Reply:
Got it, yep in that case I think this is best represented as a DOMString.

Reply:
Done, and mailed [crrev.com/c/4328291](http://crrev.com/c/4328291) on the Chromium side to make the switch.